app/src/main/java/chat/rocket/android/dagger/module/AppModule.kt

```diff
@@ -88,8 +91,8 @@ class AppModule {
 
     @Provides
     @Singleton
-    fun provideAuthTokenRepository(): AuthTokenRepository {
-        return AuthTokenRepository()
+    fun provideAuthTokenRepository(): TokenRepository {
+        return MemoryTokenRepository()
```

Comment:
Now that we are returning the `MemoryTokenRepository` we could rename the function to `provideMemoryTokenRepository()`.

Reply:
Actually I think this one is right and we should rename the others... We are hiding the implementation, so we provide a `TokenRepository`, doesn't matter for the user if it is a `MemoryTokenRepository` or a `DatabaseTokenRepository`